lib/conversion.py

```diff
@@ -158,7 +158,19 @@ def lessons_question_correct(lessons_soup):
         if 'answers' in qAttr:
             for answer in qAttr['answers']:
                 if answer['correct']:
-                    return True;
+                    return True
+
+
+def lessons_hyperlinks(lessons_soup):
+    highlight = lessons_soup.find("span", attrs={"type": "link"})
```

Comment:
I think this needs to be a more specific check. What sort of xml in lessonbuilder.xml are we matching for here, in what item types?

Reply:
We hard code the type when highlighting the vula link (for example). So when we do the soup.find, we have some degree of certainty. Hard coding done [here](https://github.com/cilt-uct/Brightspace-migrate-from-sakai/pull/16/files#diff-dd87a8940394f946ffe9811288ffa0d373cc3a54d86bfa25341d61126922e596R48).

Reply:
@smarquard, I've added a more specific check and added some custom data attributes to get us more information.